accepted/future-releases/primary-constructors/feature-specification.md

```diff
@@ -53,38 +47,26 @@ class Point {
 }
 ```
 
-A declaring constructor in the header allows us to define the same class much
-more concisely:
+A primary constructor allows us to define the same class much more concisely:
 
 ```dart
-// A declaration with the same meaning, using a declaring header constructor.
+// A declaration with the same meaning, using a primary constructor.
 class Point(var int x, var int y);
 ```
 
-A class that has a declaring header constructor cannot have any other
+A class that has a primary constructor cannot have any other
 non-redirecting generative constructors. This requirement must be upheld
-because it must be guaranteed that the declaring header constructor is
-actually executed on every newly created instance of this class. This rule
-is further motivated below.
-
-A declaring header constructor is also known as a _primary constructor_,
-because all other generative constructors must invoke the primary one
-(directly or indirectly).
-
-A declaring body constructor is slightly less concise, but it allows the
-class header to remain simpler and more readable when there are many
-parameters. The previous example would look as follows using a declaring
-body constructor:
+because it must be guaranteed that the primary constructor is executed on
+every newly created instance of this class.
 
-```dart
-// A declaration with the same meaning, using a declaring body constructor.
-class Point {
-  this(var int x, var int y);
-}
-```
+In particular, every generative constructor in a declaration that has a
```

Comment:
every *other* generative constructor

(than the primary constructor)

Reply:
Done.